module/move/willbe/src/endpoint/workspace_new.rs

```diff
@@ -0,0 +1,104 @@
+mod private
+{
+  use std::fs;
+  use std::io::Write;
+  use std::path::Path;
+  use error_tools::for_app::bail;
+  use error_tools::Result;
+  
+  /// Creates workspace template
+  pub fn workspace_new( path: &Path ) -> Result< () >
+	{
+    if fs::read_dir( path )?.count() != 0

```

Comment:
I've noticed that the current code returns an error when the designated folder does not exist. What do you think about modifying this behavior so the program automatically creates the folder if it's not found? This could potentially prevent unnecessary error returns and improve the user experience.